### ISSUE TITLE
Build fails: missing `source` directory

### ISSUE THREAD

**Issue by otheus** [jmkeyes/adsmpipe#2]
For `make PLATFORM=linux rpm`

```
    Successfully remade target file `/home/c102/c10267/projects/adsmpipe/.build/SOURCES/adsmpipe-0.1.0-1.tar.gz'.
   Finished prerequisites of target file `adsmpipe-0.1.0-1.x86_64.rpm'.
  Must remake target `adsmpipe-0.1.0-1.x86_64.rpm'.
Putting child 0x0067b3a0 (adsmpipe-0.1.0-1.x86_64.rpm) PID 3296 on the chain.
Live child 0x0067b3a0 (adsmpipe-0.1.0-1.x86_64.rpm) PID 3296
   RPM adsmpipe-0.1.0-1.x86_64.rpm
Reaping winning child 0x0067b3a0 PID 3296
Live child 0x0067b3a0 (adsmpipe-0.1.0-1.x86_64.rpm) PID 3297
Reaping losing child 0x0067b3a0 PID 3297
make[1]: *** [adsmpipe-0.1.0-1.x86_64.rpm] Error 1
Removing child 0x0067b3a0 PID 3297 from chain.
Reaping losing child 0x00c94ad0 PID 3251
make: *** [rpm] Error 2
Removing child 0x00c94ad0 PID 3251 from chain.
```

For `make PLATFORM=linux all`

```
Updating goal targets....
Considering target file `all'.
 File `all' does not exist.
  Considering target file `adsmpipe'.
   File `adsmpipe' does not exist.
    Considering target file `source/adsmpipe.o'.
     File `source/adsmpipe.o' does not exist.
      Considering target file `source/adsmpipe.c'.
       File `source/adsmpipe.c' does not exist.
        Considering target file `adsmpipe.tar.Z'.
         File `adsmpipe.tar.Z' does not exist.
         Finished prerequisites of target file `adsmpipe.tar.Z'.
        Must remake target `adsmpipe.tar.Z'.
Putting child 0x0165aaa0 (adsmpipe.tar.Z) PID 3379 on the chain.
Live child 0x0165aaa0 (adsmpipe.tar.Z) PID 3379
  WGET adsmpipe.tar.Z
Reaping winning child 0x0165aaa0 PID 3379
Live child 0x0165aaa0 (adsmpipe.tar.Z) PID 3380
Reaping winning child 0x0165aaa0 PID 3380
Removing child 0x0165aaa0 PID 3380 from chain.
        Successfully remade target file `adsmpipe.tar.Z'.
       Finished prerequisites of target file `source/adsmpipe.c'.
      Must remake target `source/adsmpipe.c'.
Putting child 0x01661870 (source/adsmpipe.c) PID 3381 on the chain.
Live child 0x01661870 (source/adsmpipe.c) PID 3381
UNPACK source/adsmpipe.c
Reaping winning child 0x01661870 PID 3381
Live child 0x01661870 (source/adsmpipe.c) PID 3382
/bin/sh: source/adsmpipe.c: No such file or directory
Reaping losing child 0x01661870 PID 3382
make[1]: *** [source/adsmpipe.c] Error 1
Removing child 0x01661870 PID 3382 from chain.
Reaping losing child 0x015caad0 PID 3348
make: *** [all] Error 2
Removing child 0x015caad0 PID 3348 from chain.
```

**Comment by otheus** [comment]
Better output:

```
UNPACK source/adsmpipe.c
Reaping winning child 0x00be8aa0 PID 3617
Live child 0x00be8aa0 (source/adsmpipe.c) PID 3618
+ tar -x -Z -f adsmpipe.tar.Z -O aix/adsmpipe/adsmpipe.c
sh: source/adsmpipe.c: No such file or directory
```

**Comment by otheus** [comment]
Your make needs to create the "source" directory somewhere.

**Comment by jmkeyes** [comment]
Fixed as of 54f8f8adce1cf69a49193bcfb64e0529d8c33eb2.
